modules/javafx.graphics/src/main/native-prism-es2/macosx/MacGLFactory.c

```diff
@@ -33,6 +33,24 @@
 #include "../PrismES2Defs.h"
 #include "com_sun_prism_es2_MacGLFactory.h"
 
+#ifdef STATIC_BUILD
+JNIEXPORT jint JNICALL
+JNI_OnLoad_prism_es2(JavaVM *vm, void * reserved) {
+fprintf(stderr, "ONLOAD PRISM_ES2\n");
```

Comment:
Maybe qualify this printf by `#if VERBOSE` or similar?

Reply:
should be fixed